src/commands/scanner/rule/add.ts

```diff
@@ -17,32 +17,34 @@ const messages = Messages.loadMessages('@salesforce/sfdx-scanner', 'add');
 
 export default class Add extends ScannerCommand {
 
-	public static description = messages.getMessage('commandDescription');
-	public static longDescription = messages.getMessage('commandDescriptionLong');
+	public static summary = messages.getMessage('commandDescription');
```

Comment:
To avoid confusion, I suggest we rename these message keys so commandDescription->commandSummary and commandDescriptionLong->commandDescription, etc. Maybe we do this in the next PR when we switch the sfdx to sf in the help.

Reply:
Yeah, my plan was to do that in the next PR, to try and limit the size of this one at least a little.